tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "allowJs": true,
     "jsx": "react",
     "moduleResolution": "node",
-    "outDir": "/",
+    "outDir": "./dist",
```

Comment:
Måtte tilpasse dette for å finne de kompilerte js-filene fra tsc slik at disse kan legges på riktig plass i prod-image't.